nativelib/src/main/scala/scala/scalanative/runtime/package.scala

```diff
@@ -55,5 +55,15 @@ package object runtime {
     GC.init()
     null
   }
+  
+  /** Useful for method like clone() */
+  private[runtime] def alloc(size: CSize): Ptr[_] = {
+    GC.malloc(size)
+  }
+  
+  /** Useful for method like clone() */
+  private[runtime] def allocAtomic(size: CSize): Ptr[_] = {
+    GC.malloc_atomic(size)
+  } 
```

Comment:
Those two helpers are trivial and don't add any value. Just use `GC.malloc` and `GC.malloc_atomic` directly.